hazelcast/src/main/java/com/hazelcast/partition/impl/InternalPartitionServiceImpl.java

```diff
@@ -1677,11 +1677,7 @@ public void dispatchEvent(PartitionEvent partitionEvent, PartitionEventListener
 
     @Override
     public String toString() {
-        StringBuilder sb = new StringBuilder("PartitionManager[" + stateVersion + "] {\n");
-        sb.append("\n");
-        sb.append("migrationQ: ").append(migrationQueue.size());
-        sb.append("\n}");
-        return sb.toString();
+        return "PartitionManager[" + stateVersion + "] {\n" + "\n" + "migrationQ: " + migrationQueue.size() + "\n}";
```

Comment:
`"] {\n" + "\n" + "migrationQ: "` could be merged.

I'm also not sure about the many newlines, since the output is just a number. Maybe this should be changed to a normally formatted `toString()` method?